### PR TITLE
EEI: clarify return and selfdestruct

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: eth_interface.md [hunk 1]
```diff
@@ -377,7 +377,9 @@ account with non-empty associated code.
 
 ## return
 
-Halt execution returning output data.
+Set the returning output data for the execution.
+
+*Note*: multiple invocations will overwrite the previous data.
```
Comment (wanderer) [inline eth_interface.md]: We could make it halt also. Is there a reason to have multiple invocations?

Comment (axic) [inline eth_interface.md]: How can you halt it?

Comment (wanderer) [inline eth_interface.md]: `throw Error('return')` I assumed that is the same way we would halt selfDestuct.

Comment (axic) [inline eth_interface.md]: That seems like a bad workaround. I think it is better to make it clear in the interface.

**Comment by axic** [comment]
@wanderer added clarifications

**Comment by wanderer** [comment]
ok lgmt

**Pull request merged**